Samples/README.md

```diff
@@ -4,29 +4,63 @@ PWMapKit Samples for iOS
 ## PhunMaps
 
 ### Overview
-- Display map with building
-- Search for points of interest in building
-- Route from current location to a selected point of interest
+- Display map with building.
+- Search for points of interest in building.
+- Route from current location to a selected point of interest.
+
+### Usage
+- Go to *./PhunMaps* directory in Terminal and run `pod update`.
+- Open the project with Xcode.
+- Fill out `applicationId`, `accessKey`, `signatureKey` and `buildingId` in *DefaultConfiguration.plist*.
+- Build the application and run it on your device.
 
 ## MapScenarios
 
 ### Overview
-- Six different use cases in both Swift and Objective C
-- Each use case functional within one ViewController
+- Six different use cases in both Swift and Objective C.
```

Comment:
More than six now, let's just drop the number to make it easier on us -> "Different use cases..."